### DISCUSSION TITLE
Provider for Bulgarian holidays

### DISCUSSION THREAD

**Discussion by pietrach** [azuyalabs/yasumi#344] [General]
I need information about holidays in Bulgaria. Will there be a dedicated provider for Bulgaria?

**Comment by stelgenhof** [comment]
Hi @pietrach At the moment there is no provider for Bulgaria. If you like, you can submit a PR for Bulgaria. Otherwise I can work on that but not sure when it will be ready.

**Reply by pietrach** [reply]
Hi @stelgenhof,
You can work on provider for Bułgaria.
I will wait.

**Reply by stelgenhof** [reply]
Hi @pietrach I've started working on the provider for Bulgaria. You can check the progress in this branch: https://github.com/azuyalabs/yasumi/tree/feature-provider-bulgaria.
